infra/gcp/terraform/k8s-infra-elections/main.tf

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+This file defines:
+- Google Project k8s-infra-elections to host the elections platform
+- Google services enabled for the project
+- GCP internal IP for the database k8s-infra-elections-db
+*/
+
+locals {
+  project_id = "k8s-infra-elections"
+}
+
+data "google_billing_account" "account" {
+  billing_account = "018801-93540E-22A20E"
+}
+
+data "google_organization" "org" {
+  domain = "kubernetes.io"
+}
+
+resource "google_project" "project" {
+  name            = local.project_id
+  project_id      = local.project_id
+  org_id          = data.google_organization.org.org_id
+  billing_account = data.google_billing_account.account.id
+}
+
+resource "google_project_service" "services" {
+  project = google_project.project.id
+
+  for_each = toset([
+    "cloudresourcemanager.googleapis.com",
+    "container.googleapis.com",
```

Comment:
This is going to cause more services to get activated, like `containerregistry`,`compute`, etc.

I've been trying to list these out in our bash.  OTOH I'm pretty sure terraform gave up trying to be authoritative about the list of services enabled for a given project